gapic-generator-cloud/Rakefile

```diff
@@ -34,4 +34,34 @@ task :ci do
   Rake::Task[:test].invoke
 end
 
+desc "Generate the expected output for tests"
+task :gen do
+  Rake::Task["gen:templates"].invoke
+end
+namespace :gen do
```

Comment:
Discussion for a later PR: Do you think it would be beneficial to expose a rake task as an interface for invoking the generator.

Reply:
👍 but I don't think it's high priority.